src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -36,13 +37,15 @@
 class PhpFileLoader extends FileLoader
 {
     protected bool $autoRegisterAliasesForSinglyImplementedInterfaces = false;
+    private static array $configurations = [];
 
     public function __construct(
         ContainerBuilder $container,
         FileLocatorInterface $locator,
         ?string $env = null,
         private ?ConfigBuilderGeneratorInterface $generator = null,
         bool $prepend = false,
+        private ?ConfigFunctionGenerator $configFunctionGenerator = null,
```

Comment:
could we find a way that doesn't require changing the constructor? can't we find a way to use the ConfigBuilderGeneratorInterface instance?